packages/cms/src/features/release-notes/release-note-entry.tsx

```
@@ -15,7 +15,7 @@ import { Button } from "@repo/ui/components/button";
 import { cn } from "@repo/ui/lib/utils";
 
 import type { ComponentReleaseNoteFieldsFragment as ReleaseNoteFields } from "../../lib/__generated/sdk";
-import { CtfRichText } from "../contentful/ctf-rich-text";
+import { CtfRichText, type EmbeddedEntryType } from "../contentful/ctf-rich-text";
```

ESLint

[error] Prefer using a top-level type-only import instead of inline type specifiers.

(import/consistent-type-specifier-style)